bring-it-up/bring-it-up
# FE#26 Use secondaryId in full service page route

## Description
#### Summary:
Updated full service page route to use secondaryId instead of serviceName.

#### Changes:
- Modified Services.tsx to use secondaryId instead of serviceName when creating routes
- Updated ServiceHoursCard and Specialties components to remove unnecessary comments and parsing of serviceId
- Added fetch in TestService to get serviceName for tabs

#### How to test this PR:
- Go to the services page and select a service.
- Check the url. It should end in the secondaryId of the service.

## Screenshots:

<img width="705" alt="Screen Shot 2023-04-03 at 7 51 03 PM" src="https://user-images.githubusercontent.com/98232527/229676507-1e0fbc03-fd60-4632-82b4-986680e0e4e0.png">

## Checklist
